### PR TITLE
Update Italian translations

### PR DESCRIPTION
**Pull request by mlocati**

### CODE REVIEW

**Comment by mlocati** [comment]
@NickeManarin What about this procedure to notify translators about new translatable strings?
- you opening an issue
- translators can subscribe to that issue
- when there are new translatable strings, you simply add a "ping" to that issue, so that translators know that there's something new to be translated

File: ScreenToGif/Resources/Localization/StringResources.it.xaml [hunk 1]
```diff
@@ -111,11 +122,26 @@
     <s:String x:Key="Interface.EditorBackground.GridSize.Apply">Applica</s:String>
 
     <s:String x:Key="Interface.Other">Altro</s:String>
-    <s:String x:Key="Interface.Other.ExtendChrome">Estendi la barra del titolo/Chrome.</s:String>
+    <s:String x:Key="Interface.Other.ExtendChrome">Estendi la barra del titolo (finestra chrome).</s:String>
```
Comment (NickeManarin) [inline ScreenToGif/Resources/Localization/StringResources.it.xaml]: Hi, "chrome" here means "window chrome" also known as "window titlebar", is it the same word/meaning in Italian?

**Comment by NickeManarin** [comment]
@mlocati I'm thinking about this idea... 

After some time, the issue entry will be buried by newer entries, isn't that a problem?

Maybe I'll could add a link to that issue on the wiki page.

File: ScreenToGif/Resources/Localization/StringResources.it.xaml (hunk 1 above)
Comment (mlocati) [inline ScreenToGif/Resources/Localization/StringResources.it.xaml]: :+1: I'll fix this translation

**Comment by mlocati** [comment]
> After some time, the issue entry will be buried by newer entries, isn't that a problem?

I'd simply add new comment to that issue as there are new translatable strings: subscribers will receive a notification, so that they know that there's something new to be translated.

The only problem I see is that translators will need a github account, but I don't think that'd be a big problem...

**Pull request merged**